jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -0,0 +1,61 @@
+job {
+  name 'terraform-deploy'
+  description('Deploy and automatically update Terraform environments upon repository changes')
+  parameters {
+    choiceParam("action", ["apply", "destroy"], "Select if you want to apply changes to environments or destroy them.")
```

Comment:
Can you put here in comments how exactly you want me to change this? Is this ok?
```
   choiceParam("action", 
                        ["apply", "destroy"], 
                        "Select if you want to apply changes to environments or" 
                       + "destroy them.")
```
or
```
   choiceParam("action", ["apply", "destroy"], 
                        "Select if you want to apply changes to environments or" 
                       + "destroy them.")
``` ?